lonelyplanet/bane
# Use new Rizzo mixin to show type icons in search results

> Rizzo PR: https://github.com/lonelyplanet/rizzo/pull/1306

## Description

1. When user presses enter key in search bar, user is redirected to the search page with the current text in the input field
2. The icon now represents the specific type of search result

## Verification

1. Enter in a Tier 1 city that will return places, activities,etc (e.g. London)
2. Verify that the appropriate icon is on the corresponding background image for the results
